### PR TITLE
[MIG][15.0]project_task_code: migration 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: project_task_code/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sequential Code for Tasks",
+    "version": "15.0.0.0.1",
```
Comment (flotho) [inline project_task_code/__manifest__.py]: Hi, thanks for your contribution.
I'm not sure that the version number follow the appropriate pattern.
Should be 15.0.1.0.1 IMHO.

**Comment by flotho** [comment]
Additionally, did you preserve history?

**Comment by bizzappdev** [comment]
> Additionally, did you preserve history?

Yes, History has been respected.

**Review by lamtritin90 — APPROVED** [review]
LGTM

**Review by flotho — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**